scripts/main.js

```diff
@@ -0,0 +1,164 @@
+// main.js
+
+const allSquares = elementsByClassName("square");
+
+function main() {
+    allSquares
+        .forEach(addGameClickListeners);
+
```

Comment:
space here?

Reply:
I personally prefer currying methods that works in a chain in a new indentation level